### ISSUE TITLE
Make right slide ask for group deletion, and add background colors

### ISSUE THREAD

**Issue by davidhedlund** [douzifly/clear-todolist#32]
I can add groups, but it is not possible to remove them. Can you add a waste basket icon next to the groups?

**Comment by inlytica** [comment]
Yes!!  Extremely annoying.  How was this missed during development.

**Comment by rbalogic** [comment]
I am also facing this. Please check into it.

**Comment by ghost** [comment]
It is possible to delete groups but it is hidden. Go to Settings>Edit Group. Hold and slide left on any group name that is not the current group you are currently in. Click on trash can.

**Comment by davidhedlund** [comment]
@Whispergod Thanks.

> It is possible to delete groups but it is hidden. Go to Settings>Edit Group.

It could be done in the group window outside Edit Group as well.

@douzifly 
Left slide will ask for group deletion but not right slide like for task deletion. I suggest a unified command by setting right slide for group deletion, left slide can be kept  to do this as well.

**Comment by ghost** [comment]
@davidhedlund Were you waiting a year to hear my answer? I actually just got this app today.

**Comment by davidhedlund** [comment]
@Whispergod 
> Were you waiting a year to hear my answer?

Yes, but I'm not alone. =)

**Comment by douzifly** [comment]
LOL

**Comment by davidhedlund** [comment]
@douzifly Another reason why it's hard to figure out why it's possible to delete groups is because the group entries lack background colors. The group list has a plain white background.
